### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

**Review by Menselface — APPROVED** [review]
Looks fine

File: app/main.py [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+def copy_file(command: str) -> None:
+    command_check = match(r"cp .*\.txt .*\.txt", command)
+
+    if command_check:
+        first_file_name = command.split()[1]
+        second_file_name = command.split()[2]
+
+        if first_file_name != second_file_name:
+            with (open(first_file_name, "r") as first_file,
```
Comment (Y-Havryliv) [inline app/main.py:12]: `first_file` and `second_file` are not really meaningful

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,15 @@
-# write your code here
+from re import match
+
+
+def copy_file(command: str) -> None:
+    command_check = match(r"cp .*\.txt .*\.txt", command)
```
[the remark this answers is not in the document]
Comment (Vasyl-Poremchuk) [reply app/main.py]: I thought at first that the file format should be checked as well, but then I realized that the previous check only checks files with `.txt`  format, which is not the general case. Thanks for the comment)

**Review by Y-Havryliv — APPROVED** [review]
Well done!

**Pull request closed without merging**